website/docs/prysm-usage/slasher.md

```diff
@@ -3,71 +3,41 @@ id: slasher
 title: Running a slasher
 sidebar_label: Running a slasher
 ---
-This section provides instructions to run [slasher](https://github.com/prysmaticlabs/prysm/tree/master/slasher) to identify and report slashable offenses on eth2 .  If slasher detects a slashable offense, proof is sent to the [beacon-chain node](https://docs.prylabs.network/docs/how-prysm-works/beacon-node/) for inclusion in a block.  [Validators](https://docs.prylabs.network/docs/how-prysm-works/prysm-validator-client/) earn a small whistleblower reward for including this proof into a block.  
+This section provides instructions to run [slasher](https://github.com/prysmaticlabs/prysm/tree/master/slasher) as an **optional** process to report slashable offenses on eth2 .  If slasher detects a slashable offense, proof is sent to the [beacon-chain node](https://docs.prylabs.network/docs/how-prysm-works/beacon-node/) for inclusion in a block.  [Validators](https://docs.prylabs.network/docs/how-prysm-works/prysm-validator-client/) earn a small whistleblower reward for including this proof into a block.  
 
-:::danger Slasher Can Be a Resource Hog
-Slasher can be a huge resource hog during times of no chain finality, which can manifest as massive RAM usage. Please make sure you understand the risks of this, especially if you want high uptime for your beacon nodes. Slasher places significant stress on beacon nodes when the chain has no finality, and might be the reason why your validators are underperforming if your beacon node is under this much stress.
+:::tip Slasher Requires Significant Disk Space
+Slasher is essentially a beacon node with **superpowers**. It might use over 70Gb of disk space mainnet today, and we recommend running a slasher with more than the minimal specifications in our [installation guides](/docs/install/install-with-script) and instead running the **recommended** specifications.
```

Comment:
```suggestion
Slasher is essentially a beacon node with **superpowers**. It might use over 70Gb of disk space on mainnet today, and we recommend running a slasher with more than the minimal specifications in our [installation guides](/docs/install/install-with-script) and instead running the **recommended** specifications.
```